src/components/WwwFrame/TheHeader.vue

```diff
@@ -1,248 +1,252 @@
 <template>
-	<header class="top-nav">
-		<div class="header-row row">
-			<router-link class="header-logo header-button" to="/" v-kv-track-event="['TopNav','click-Logo']">
-				<kv-icon name="new-kiva-logo" />
-			</router-link>
-			<router-link
-				:id="lendMenuId"
-				to="/lend"
-				class="header-button"
-				v-kv-track-event="['TopNav','click-Lend']"
-			>
-				<span>Lend <kv-icon name="triangle" /></span>
-			</router-link>
-			<button class="search-toggler header-button"
-				:class="{'show-for-large': isVisitor}"
-				:aria-expanded="searchOpen ? 'true' : 'false'"
-				:aria-pressed="searchOpen ? 'true' : 'false'"
-				aria-controls="top-nav-search-area"
-				@click="toggleSearch"
-				v-kv-track-event="['TopNav','click-search-toggle']"
-			>
-				<kv-icon v-show="!searchOpen" class="search-icon" name="magnify-glass" />
-				<kv-icon v-show="searchOpen" class="close-icon" name="x" />
-			</button>
-			<div class="flexible-center-area">
-				<div id="top-nav-search-area" :aria-hidden="searchOpen ? 'false' : 'true'">
-					<button class="close-search hide-for-large"
-						:aria-expanded="searchOpen ? 'true' : 'false'"
-						:aria-pressed="searchOpen ? 'true' : 'false'"
-						aria-controls="top-nav-search-area"
-						@click="toggleSearch"
-						v-kv-track-event="['TopNav','click-search-close-mobile']"
-					>
-						<kv-icon class="close-icon" name="x" />
-					</button>
-					<search-bar ref="search" />
+	<div>
+		<appeal-banner v-if="showAppealBanner"/>
```

Comment:
I think it would be cleaner to include the appeal-banner component in the WwwPage component instead of TheHeader. Additionally, this v-if could be placed on the root element of the AppealBanner component, so then the checks for whether it should be displayed can be entirely contained within that component.